### DISCUSSION TITLE
GluonTS library

### DISCUSSION THREAD

**Discussion by fsebastian93** [PaddlePaddle/PaddleSpatial#75] [BaiduKDDCup2022] [answered]
Hi everyone! Please if is it possible to install that library in the base framework? Thanks so much.

**Comment by fersebas** [comment, marked as answer]
Please?

**Reply by ramber1836** [reply]
It has been added to the base environment. Please refer to the manual for the latest list of preinstalled packages in different environments. Thanks.

**Reply by fersebas** [reply]
Sorry, I see. But it needs mxnet installed before. Sorry about that.
https://ts.gluon.ai/install.html
![image](https://user-images.githubusercontent.com/49498032/170857954-5d9d4fd5-8488-480e-9f99-8acde6e90d30.png)

**Comment by ramber1836** [comment]
This has been supported in the base framework.

Regards,
Xinjiang Lu
